demos/include/iot_demo_runner.h

```diff
@@ -78,6 +78,15 @@
         #undef democonfigDEMO_PRIORITY
         #define democonfigDEMO_PRIORITY     democonfigMQTT_ECHO_TASK_PRIORITY
     #endif
+#elif defined( CONFIG_CORE_MQTT_CONNECTION_SHARING_DEMO_ENABLED )
+    #define DEMO_entryFUNCTION              RunCoreMqttConnectionSharingDemo
+    #if defined( democonfigMQTT_ECHO_TASK_STACK_SIZE )
+        #undef democonfigDEMO_STACKSIZE
+        #define democonfigDEMO_STACKSIZE    democonfigMQTT_ECHO_TASK_STACK_SIZE
+    #endif
+    /* This demo must have priority at least one greater than idle. */
+    #undef democonfigDEMO_PRIORITY
+    #define democonfigDEMO_PRIORITY         ( tskIDLE_PRIORITY + 1 )
```

Comment:
Should this customer configurable?

Reply:
Ideally yes, I didn't see a way to do it without creating a new `democonfigCONNECTION_SHARING_DEMO_TASK_PRIORITY` macro in `aws_demo_config.h` for all of the boards. Should I do that?

Reply:
It might help in the future for certain boards that need different priorities

Reply:
Updated